davidia/server/plotserver.py

```diff
@@ -455,40 +554,53 @@ async def prepare_data(self, msg: PlotMessage, omit_client: PlotClient = None):
             if c is not omit_client:
                 await c.add_message(new_msg)
 
+    def get_clients_from_uuid(self, uuid: str):
+        return [c for cl in self._clients.values() for c in cl if c.uuid == uuid]
```

Comment:
Can improve this to be a generator expression and combine L219-220. Rename as `clients_with_uuid`